app/scripts/components/analysis/define/index.tsx

```diff
@@ -105,6 +105,22 @@ export default function Analysis() {
   const thematic = useThematicArea();
   if (!thematic) throw resourceNotFound();
 
+  useEffect(() => {
+    // Generate enough thresholds to get regular updates from the observer
+    const threshold = Array(300).fill(0).map((_, i) => i/300);
+    const obs = new IntersectionObserver(() => {
+      window.tick?.();
+    }, {
+      threshold
+    });
+
+    // TODO use a ref on PageMainContent
+    if (document.getElementsByTagName('main')[0]) {
+      obs.observe(document.getElementsByTagName('main')[0]);
+    }
+
+  }, []);
```

Comment:
It is a bit confusing that this can't go inside `app/scripts/utils/use-sliding-sticky-header.ts`, since when that mounts the `main` is not yet available.